src/wasm/kotlin/java/arcs/Addressable.kt

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
```

Comment:
I think it would be best to have the same copyright comment as what I've got in src_kt and shells/android

Reply:
The difference is whitespace? Are there content differences?

Reply:
Whitespace, as well as no `@license`.   You can configure Android Studio to automatically add the copyright to all new files.

![image](https://user-images.githubusercontent.com/135470/69772835-ddcf7e00-1145-11ea-9028-4a0dd9a55c33.png)

![image](https://user-images.githubusercontent.com/135470/69772846-e7f17c80-1145-11ea-93ce-71e436925b0e.png)